Source/WebKit/UIProcess/mac/PageClientImplMac.h

```diff
@@ -171,6 +171,7 @@ class PageClientImpl final : public PageClientImplCocoa
     void didFirstLayerFlush(const LayerTreeContext&) override;
 
     RefPtr<ViewSnapshot> takeViewSnapshot(std::optional<WebCore::IntRect>&&) override;
+    RefPtr<ViewSnapshot> takeViewSnapshotAlwaysUsingNonHardwareCapturePath(std::optional<WebCore::IntRect>&&) override;
```

Comment:
Why is this a separate method, and not reusing the flag we already have?

Reply:
I did this to avoid having to change multiple `.h` files that also overrides `takeViewSnapshot` (like `PageImplIOS.h`).

Reply:
Or I guess we could've also made an override that takes two arguments. Hmm, maybe that's better actually.